tests/app/test_app.py

```diff
@@ -18,10 +17,13 @@
 from secure_message.api_mocks.party_service_mock import PartyServiceMock
 
 
-class FlaskTestCase(unittest.TestCase):
+class AppTestCase(unittest.TestCase):
     """Test case for application endpoints"""
 
     BRES_SURVEY = "33333333-22222-3333-4444-88dc018a1a4c"
+    SPECIFIC_INTERNAL_USER = "SpecificInternalUserId"
```

Comment:
Can this not be a real uuid as well if we're aliasing it anyway?

Reply:
It lines up with the values in the mocks and leaving them human readable makes it easier while debugging